### PR TITLE
Allow bylines on CSP pages

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #N/A

**Overall change:**
- Allow Bylines to render for both CSP and STY assets

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by chris-hinds — COMMENTED** [review]
Ignore this comment..... I commented on the wrong PR and cant delete it :(

**Review by joshcoventry — APPROVED** [review]

File: src/app/routes/cpsAsset/getInitialData/addBylineBlock/index.test.js [hunk 1]
```diff
@@ -142,6 +142,13 @@ describe('addBylineBlock', () => {
     expect(addBylineBlock(input)).toEqual(input);
   });
 
+  it('should return input if page type is not CSP', () => {
+    const input = deepClone(styInput);
```
Comment (andrewscfc) [inline src/app/routes/cpsAsset/getInitialData/addBylineBlock/index.test.js]: This test is an exact duplicate of the one above, shouldn't there be an additional test to check a byline is added for CSP pages?

Comment (chris-hinds) [reply src/app/routes/cpsAsset/getInitialData/addBylineBlock/index.test.js]: let me look into this now.

Comment (chris-hinds) [reply src/app/routes/cpsAsset/getInitialData/addBylineBlock/index.test.js]: @andrewscfc I have added a new test testing that a byline is also added for csp assets

**Review by andrewscfc — APPROVED** [review]
Thanks for the test @chris-hinds

**Pull request merged**